apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -137,8 +144,8 @@ export default connect(
     onPickNewAnimation() {
       dispatch(pickNewAnimation());
     },
-    onPickLibraryAnimation(animation) {
-      dispatch(pickLibraryAnimation(animation));
+    onPickLibraryAnimation(animation, multiselect) {
```

Comment:
nit (here and elsewhere): it's a little hard to tell the type/expected value of `multiselect`, especially since it gets passed through so many places. There's a helpful comment in the action creater, but it takes kind of a lot of tracing through the code to get there. Maybe a name like `isMultiSelectEnabled` would be more clear if you're just looking at the caller?